ui/src/literature/components/AssignLiteratureItemDrawer.tsx

```diff
@@ -70,6 +45,13 @@ function AssignLiteratureItemDrawer({
   paperId,
 }: AssignLiteratureItemDrawerProps) {
   const [selectedAuthorId, setSelectedAuthorId] = useState<string>();
+  const [availableAuthors, setAvailableAuthors] = useState<IAuthorResult[]>([]);
+  
+  useEffect(() => {
+    // @ts-ignore
+    setAvailableAuthors(authors.toArray());
```

Comment:
why we need an array?

Reply:
can we just read authors from the state directly?

Reply:
Antd component `List` that I use for displaying list with pagination requires array of elements as input